lib/mindwendel_web/templates/error/error_page.html.heex

```diff
@@ -8,6 +8,7 @@
       content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"
     />
     <title>mindwendel</title>
+    <%!-- TODO: ~p isn't known/used here and so also fails, but I'm also not familiar with this page --%>
```

Comment:
I think it's still being called from a view and views don't have sigil available? Best way would be to refactor errors into components I think so we can remove all view files.

Reply:
Agreed. I'm not sure though - I think views should also have ~p available, it's probably just the matter of a missing import.

Reply:
@JannikStreek yup https://hexdocs.pm/phoenix/Phoenix.VerifiedRoutes.html#module-options

`use Phoenix.VerifiedRoutes` should solve the problem

Reply:
We should still move it to components overall but maybe that's a stepping stone :)